plugins/fishlim/plugin_hexchat.c

```diff
@@ -98,6 +100,33 @@ static hexchat_context *find_context_on_network (const char *name) {
     return ret;
 }
 
+/**
+ * Retrive the prefix character for own nick in current context
+ * @return @ or + or NULL
+ */
+const char *get_my_own_prefix(void) {
+    const char *own_nick;
+    hexchat_list* list;
+
+    /* Display message */
+    own_nick = hexchat_get_info(ph, "nick");
+
+    if (!own_nick)
+        return NULL;
+
+    /* Get prefix for own nick if any */
+    list = hexchat_list_get (ph, "users");
+    if (list) {
+        while (hexchat_list_next(ph, list)) {
+            if (strcmp(own_nick, hexchat_list_str(ph, list, "nick")) == 0)
+                return hexchat_list_str(ph, list, "prefix");
```

Comment:
`list` leaks.

Reply:
Also just `g_strdup()` this value. I don't like returning unclearly owned strings.